src/apps/mux/helpers/common.ts

```diff
@@ -0,0 +1,150 @@
+import Axios, { AxiosInstance } from 'axios';
+import { ethers } from 'ethers';
+
+import { IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { MuxContractFactory } from '~apps/mux';
+import { LiquidityAsset } from '~apps/mux/helpers/mux.mlp.token-helper';
+import { ContractType } from '~position/contract.interface';
+import { BaseToken } from '~position/token.interface';
+import { Network } from '~types/network.interface';
+
+interface ReaderAssets {
+  symbol: string;
+  tokenAddress: string;
+  decimals: number;
+  isStable: boolean;
+  isTradable: boolean;
+  isOpenable: boolean;
+  useStableTokenForProfit: boolean;
+  isEnabled: boolean;
+}
+
+export const DECIMALS = 18;
+export const RATIO_DECIMALS = 5;
+export const ASSET_IS_STABLE = 0x00000000000001; // is a usdt, usdc, ...
+export const ASSET_CAN_ADD_REMOVE_LIQUIDITY = 0x00000000000002; // can call addLiquidity and removeLiquidity with this token
+export const ASSET_IS_TRADABLE = 0x00000000000100; // allowed to be assetId
+export const ASSET_IS_OPENABLE = 0x00000000010000; // can open position
+export const ASSET_IS_SHORTABLE = 0x00000001000000; // allow shorting this asset
+export const ASSET_USE_STABLE_TOKEN_FOR_PROFIT = 0x00000100000000; // take profit will get stable coin
+export const ASSET_IS_ENABLED = 0x00010000000000; // allowed to be assetId and collateralId
+export const ASSET_IS_STRICT_STABLE = 0x01000000000000; // assetPrice is always 1 unless volatility exceeds strictStableDeviation
+
+const invalidAddress = '0x0000000000000000000000000000000000000000';
```

Comment:
Instead of redefining this constant, you can use our global constant named ZERO_ADDRESS